src/network/C4NetIO.cpp

```diff
@@ -3923,9 +3901,7 @@ void C4NetIOUDP::DebugLogPkt(bool fOut, const C4NetIOPacket &Pkt)
 // *** C4NetIOMan
 
 C4NetIOMan::C4NetIOMan()
-		: StdSchedulerThread(),
-		iNetIOCnt(0), iNetIOCapacity(0),
-		ppNetIO(nullptr)
+		: StdSchedulerThread()
```

Comment:
The base ctor is called anyway and doesn't need to be listed explicitly; that way this entire function can be defaulted too.